src/charts/donut/donut-pct-chart-directive.js

```diff
@@ -305,7 +307,7 @@ angular.module('patternfly.charts').directive('pfDonutPctChart', function (c3Cha
 
           // default to 'used' info.
           centerLabelText = { bigText: $scope.data.used,
-                              smText:  $scope.config.units + ' Used' };
+                              smText:  $i18next('pfDonutPctChart.used', { units: $scope.config.units }) };
```

Comment:
Yes they may potentially want to change it per chart but doing it this way also gives translators the context of what they are translating. Without the directive as a prefix they may have issues figuring out where the string is actually being used.